hl7/4.2/modules/ROOT/pages/hl7-connector-reference.adoc

```diff
@@ -176,86 +237,119 @@
 
 == Types
 
-[[Reconnection]]
-=== Reconnection
+[[ExpirationPolicy]]
+=== Expiration Policy
+
+Configures the maximum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration.
 
 [%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | The reconnection strategy to use. |  |
+| Max Idle Time a| Number | Maximum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. |  |
+| Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | A time unit that qualifies the *Max Idle Time* field. |  |
 |===
 
-[[reconnect]]
-=== Reconnect
+[[forever]]
```

Comment:
Reconnect Forever instead of Forever? And is there a table missing below it?

Reply:
Forever is correct.